release/restcomm-assembly-as7/pom.xml

```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.restcomm</groupId>
+        <artifactId>assembly-parent</artifactId>
+        <version>9.0.0-SNAPSHOT</version>
+    </parent>
+    <groupId>org.restcomm</groupId>
+    <artifactId>assembly-as7</artifactId>
+    <packaging>pom</packaging>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <sipservletapi.version>3.2.0-89</sipservletapi.version>
+        <docker.buildArg.CONNECT_VERSION>${project.version}</docker.buildArg.CONNECT_VERSION>
+    </properties>
+    <profiles>
+        <profile>
+            <id>all-in-one</id>
```

Comment:
what happens on maven:deploy when both profiles are enabled. Will both binaries will get deployed into nexus with different classifier?

Reply:
@jaimecasero with regards to the docker images, that is not an issue, because we have `restcomm/restcomm` and `restcomm/connect` images. 

What would you propose for WAR binaries?

Reply:
I guess you mean ZIP binaries... The assemble ZIP files should be deployed in nexus as attachment of project with specific classifier. But since you are using same project coordiante to create slightly different ZIP binaries, then assembly classifier is required to be different...

Honestly i tend to think these are really two different maven artifacts (all-in-one vs standalone),which should be modelled as separate maven modules with diff coordiantes(artifactid). In terms of zip versus docker, im fine sharing same project module, as their are semantically the same binary file (same features delivered) but with different flavors (ZIP versus Docker image). Also, we dont want to deploy docker image into Nexus as maven dependency,but be deployed in Nexus docker registry as docker image...

Again, im not sure about valid usage of profiles here...